app/src/main/kotlin/com/hippo/ehviewer/ui/reader/SliderPagerDoubleSync.kt

```diff
@@ -12,7 +12,7 @@ import androidx.compose.runtime.snapshotFlow
 import kotlinx.coroutines.flow.collectLatest
 
 class SliderPagerDoubleSync(private val lazyListState: LazyListState) {
-    private var sliderFollowPager by mutableStateOf(false)
+    private var sliderFollowPager by mutableStateOf(true)
```

Comment:
`info.visibleItemsInfo` is empty at initial state.

Reply:
Done